arroyo-rpc/src/lib.rs

```diff
@@ -58,6 +58,10 @@ pub enum ControlResp {
         task_index: usize,
         error: String,
     },
+    Error {
+        message: String,
```

Comment:
Should this also include the operation_id and task index?

Reply:
Do you think those should be saved to the db and shown in the UI too?

Reply:
I think at least knowing the operator will be useful (in some cases it may be obvious, but in others it will not be -- imagine that the error is that the message failed to deserialize, but you have multiple sources)

Reply:
Ok, I added both to the database